server/http_hooks.go

```diff
@@ -45,3 +51,62 @@ func (p *Plugin) handleStatus(w http.ResponseWriter, r *http.Request) {
 func (p *Plugin) handleHello(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("Hello World!"))
 }
+
+func (p *Plugin) handleEphemeralUpdate(w http.ResponseWriter, r *http.Request) {
+	request := model.PostActionIntegrationRequestFromJson(r.Body)
+	URL := fmt.Sprintf("%s", *p.API.GetConfig().ServiceSettings.SiteURL)
+
+	count := request.Context["count"].(float64) + 1
+
+	post := &model.Post{
+		Id:        request.PostId,
+		ChannelId: request.ChannelId,
+		UserId:    request.UserId,
+		Message:   "updated ephemeral action",
+		Props: model.StringInterface{
+			"attachments": []*model.SlackAttachment{
+				{
+					Actions: []*model.PostAction{
+						{
+							Integration: &model.PostActionIntegration{
+								Context: model.StringInterface{
+									"count": count,
+								},
+								URL: fmt.Sprintf("%s/plugins/%s/ephemeral/update", URL, manifest.Id),
+							},
+							Type: model.POST_ACTION_TYPE_BUTTON,
+							Name: fmt.Sprintf("Update %d", int(count)),
+						},
+						{
+							Integration: &model.PostActionIntegration{
+								Context: model.StringInterface{},
+								URL:     fmt.Sprintf("%s/plugins/%s/ephemeral/delete", URL, manifest.Id),
+							},
+							Type: model.POST_ACTION_TYPE_BUTTON,
+							Name: "Delete",
+						},
+					},
+				},
+			},
+		},
+	}
+	p.API.UpdateEphemeralPost(request.UserId, post)
+
+	resp := &model.PostActionIntegrationResponse{}
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write(resp.ToJson())
```

Comment:
Is this need? Isn't an empty response enough?

Reply:
Without this response errors appear in the log `Action integration error                      caller="mlog/log.go:172" err_details="err=EOF" err_where=DoPostAction http_code=400 ip_addr="::1" method=POST path=/api/v4/posts/5qz78hib7idpzdo9fun5i4jypo/actions/74377u6mfbbk9kr38orwu1319w request_id=njzege7kcjrqicw7mfbnkzende user_id=mywge9a3ibyd9nosbkaaubo4ph`

Reply:
Didn't know that. Thanks!